tsconfig.json

```
@@ -1,14 +1,5 @@
 {
-  "compilerOptions": {
-    "composite": true,
-    "declaration": true,
-    "declarationMap": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "noEmit": true
-  },
+  "extends": "./tsconfig.options.json",
```

```suggestion
  "extends": "./tsconfig.options.json",
```

Perhaps the main `tsconfig.json` we reserve for referencing the projects only?